### ISSUE TITLE
Receiving reload command before hello results in ProtocolError

### ISSUE THREAD

**Issue by smhg** [napcs/node-livereload#115]
## What version are you using?
0.6.3

## Expected result
Ideally no reload command before a hello command.

## Actual result
> LiveReload protocol error (invalid command 'reload', only valid commands are: hello)) after receiving data: "{"command":"reload","path":"/home/.......","liveCSS":true,"originalPath":"","overrideURL":""}".

![reload-hello](https://user-images.githubusercontent.com/2411348/34466920-a565cf78-eee3-11e7-8d4d-1adbdabf7d93.png)

## Steps to reproduce issue
I run livereload as an NPM script. Meaning this is in my package.json:
```json
"scripts": {
  "watch": "watchify ... & node task/watch.js ... & livereload public"
}
```
(FWIW, it's one watch for JS and one for CSS)
I'm pretty sure it only happens when JS files change. Those builds take a bit of time (about 1-2 seconds). I suspect it happens around the time the old file is 'deleted' and before the new file is written. Just guessing, but this might cause 2 subsequent reload events?

## Why is this important?
If it is something that can be avoided inside the livereload server, I guess it is good to keep the console clean.

**Comment by smhg** [comment]
For others running into this: the actual error in the Chrome console is `ProtocolError is not defined` (on line 345 of ext/livereload.js). Which seems like a separate real bug, but this hides the deeper error above.

**Comment by smhg** [comment]
Closing because it is fixed in master.
@napcs would you be willing to release this as 0.6.4?

**Comment by napcs** [comment]
@smhg I have released 0.7.0 now with this fix and several others.
